src/css/profile/mobile/common/popup.less

```diff
@@ -93,6 +88,144 @@
 	}
 }
 
+@media (min-width: 0px) and (max-width: 479px) {
+	@media (min-height: 600px) and (max-height: 959px) {
+		.ui-popup {
+			width: 100%;
+			margin: 0 auto;
+			background-color: rgba(255, 255, 255, 0);
+			margin-top: 40px;
+			margin-bottom: 16px;
+			padding-top: 16px;
+			.ui-popup-header {
+				font-size: 20px;
+				text-align: left;
+				padding: 24px 24px 18px 24px;
+				position: sticky;
+			}
+			.ui-popup-content {
+				font-size: 16px;
+				text-align: left;
+				padding: 0px 24px 24px 24px;
+				.ui-listview {
+					border-top-width: 1px;
+					border-top-color: #CCCCCC;
+					border-top-style: solid;
+					border-bottom-width: 1px;
+					border-bottom-color: #CCCCCC;
+					border-bottom-style: solid;
+				}
+			}
+			.ui-popup-footer {
+				padding: 0px 24px 20px 24px;
+				position: sticky;
+			}
+		}
+	}
+}
+
+@media (min-width: 600px) and (max-width: 959px) {
+	@media (min-height: 0px) and (max-height: 479px) {
+	  	.ui-popup {
+			width: 60%;
+			margin: 0 auto;
+			background-color: rgba(255, 255, 255, 0);
```

Comment:
Can we define it in the light.variables.less or dark.variables.less?

Reply:
Background-color for the border of the popup. Use it to simply look transparent. 
It's not the color inside the popup.